src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/LineWrappingHandler.java

```diff
@@ -169,35 +151,83 @@ public void checkIndentation(DetailAST firstNode, DetailAST lastNode, int indent
         else {
             firstNodeIndent = startIndent;
         }
-        final int currentIndent = firstNodeIndent + indentLevel;
-
         for (DetailAST node : firstNodesOnLines.values()) {
-            final int currentType = node.getType();
+            // In arrays arrayIndentation is used
```

Comment:
`In arrays arrayIndentation is used`->`ArrayInitIndentation is used for array elements`

Reply:
Done